ui/app/AppLayouts/Profile/panels/SupportedTokenListsPanel.qml

```diff
@@ -89,6 +90,7 @@ StatusListView {
             readonly property TokenListPopup popup: TokenListPopup {
                 parent: root
 
+                sourceKey: delegate.key
```

Comment:
What's that key? Ideally role `key` should be used only for unique identification, with no other assumptions about that. So it should not be displayed because it may change what's the format of that role.

Reply:
What I wanted to indicate is that this `delegate.key` is finally bound to `headline.title` what's violation of the rule I mentioned above.